### ISSUE TITLE
Notification Badge

### ISSUE THREAD

**Issue by bkonia** [jpwalters/OpenVBX-Plugin-SMS-Conversation#24]
It's annoying to have to constantly click on the SMS inbox to clear the notification badge. It would be great if there was an option to move the badge from the regular SMS inbox to SMS Conversations.

**Comment by jpwalters** [comment]
Added a notification badge to the sms conversation plugin might be possible.  Modifying the behavior of the inbox badge will likely not be possible since that part of OpenVBX but I'll take a look and see what can be done.

Note: When you click a conversation in the plugin it will mark the message in the inbox as read and when the page is reloaded the notification will disappear on the indox.

**Comment by bkonia** [comment]
​I'd think you could hide the Inbox badge using CSS.​
ᐧ

**Comment by jpwalters** [comment]
Yes, it likely can.  I'm also mentally debating if this plugin should really alter the functionality of "another plugin".

**Comment by gegere** [comment]
Remember KISS as you improve and build this plugin.

**Comment by bkonia** [comment]
Desktop notifications would also be awesome! Here's a JS library that makes it easy:

https://github.com/alexgibson/notify.js/

**Comment by jpwalters** [comment]
Yeah that could be very nice.  I'll checkout the project.
